### PR TITLE
Update Foreman Gem dependencies

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -243,7 +243,6 @@ BuildRequires: %{?scl_prefix}rubygem(jwt) < 2.3.0
 BuildRequires: %{?scl_prefix}rubygem(graphql) >= 1.8.0
 BuildRequires: %{?scl_prefix}rubygem(graphql) < 1.9.0
 BuildRequires: %{?scl_prefix}rubygem(graphql-batch)
-BuildRequires: %{?scl_prefix}rubygem-activerecord-nulldb-adapter
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec:246]: Perhaps move this out of the generated part and add a comment that it's from the nulldb group?

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
Sprockets::FileNotFound: couldn't find file 'jquery.ui.spinner' with type 'application/javascript'


Well, the source release with the new code is not there yet :(

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
This seems to build now.

@ekohl should we ignore the direction of https://github.com/theforeman/foreman/pull/7605 and merge this regardless? And then update once more when we decide to do smth else in core?

**Comment by ekohl** [comment]
Repoclosure is still unhappy though.
```
[2020-04-23T11:14:07.564Z]     package: foreman-assets-2.1.0-0.14.develop.20200423110443git77c2ee8.el7.noarch from downloaded_rpms
[2020-04-23T11:14:07.564Z]       unresolved deps:
[2020-04-23T11:14:07.564Z]         tfm-rubygem(jquery-ui-rails) >= 6.0
```

**Comment by evgeni** [comment]
That's expected, it wasn't mashed yet

**Pull request merged**